### DISCUSSION TITLE
Check if block was actually placed

### DISCUSSION THREAD

**Discussion by VendoAU** [Minestom/Minestom#999] [Help] [answered]
When the player is too close to actually place a block the PlayerBlockPlaceEvent is still called.
Is there a way to check if a block was actually placed?

**Comment by Eoghanmc22** [comment, marked as answer]
PlayerBlockPlaceEvent is called only if the collision check passes.
see https://github.com/Minestom/Minestom/blob/afddb6f549dd37d7b1d8c1dea4993cb09061db30/src/main/java/net/minestom/server/listener/BlockPlacementListener.java#L109-L128

**Reply by VendoAU** [reply]
Ok, yeah I'm dumb.
For some reason I thought I had tested this, but apparently not.
